src/app/questionnaire/item-hint/item-hint.component.scss

```diff
@@ -0,0 +1,13 @@
+.learn-more-container {
+  display: flex;
+  mat-icon {
+    margin-right: 0.2rem;
+  }
+  a {
+    margin-right: 0.2rem;
+    color: inherit;
+    &:hover {
```

Comment:
Add pseudo classes :active und :focus to ensure it is highlighted on tabbing through the elements.